src/utils.ts

```diff
@@ -64,7 +64,8 @@ export async function postSlackNotification (
     if (payloadForPushes?.compare !== undefined) {
       const beforeSha = payloadForPushes.before.slice(0, 7)
       const afterShaMessage = payloadForPushes.head_commit.message ?? ''
-      commitText = `<${payloadForPushes.compare}|${beforeSha} ⇢ ${afterSha} ${afterShaMessage}>`
+      const shortShaMessage = trimEllipsis(afterShaMessage.replace(/(\r\n|\n|\r).*$/gm, ''), 60) // keep only some first symbols of the first line
```

Comment:
How did you pick 60? I think 80 characters is a common recommended git commit message limit but not completely sure 🤔

Reply:
So google says it’s 72 chars, but I thought of having shorter one to reduce chat spam. We can adjust that as we go if that will be annoying.